boa3_test/test_sc/interop_test/storage/StorageGetCheck.py

```diff
@@ -0,0 +1,12 @@
+from boa3.builtin.compile_time import public
+from boa3.sc.storage import try_get_int, put_int
+
+
+@public
+def put_value(key: bytes, value: int):
+    put_int(key, value)
+
+
+@public
+def get_value(key: bytes) -> tuple[int, bool]:
+    return try_get_int(key)
```

Comment:
I think it would be good to have another test that uses the storage value and boolean individually, like `try_get_int(key)[0]` and `try_get_int(key)[1]`